src/templates/partials/profile-detail.twig

```diff
@@ -1,142 +1,157 @@
+          </dd>
+          <dt class="font-semibold border-top">Office Hours</dt>
+          <dd class="mb-md-3">Tuesdays 3-4pm<br>Thursdays 11am-12pm</dd>
+        </dl>
+      </div>
+
+    </div>
+
+  {% endblock %}
+  {% block content %}
+    {% set links = random([['Links'], ['Social Links'], ['Links', 'Social Links']]) %}
+    {% set departments = ['Biology', 'Dean of Students', 'Head of Sciences'] %}
+    <dl class="f2 mt-md-2 mt-lg-0 profile-detail__programs">
+      <dt class="font-semibold">Additional Programs</dt>
+      <dd>
+        {% for department in departments %}
+          <span>
+            <a href="#">{{department}}</a>
+          </span>
+        {% endfor %}
+      </dd>
+    </dl>
+
+    <div class="typography mb-5">
+      <p>Pellentesque nibh felis, eleifend id, commodo in, interdum vitae, leo. Praesent eu elit. Ut eu ligula. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos hymenaeos. Maecenas elementum augue nec nisl. Proinauctor lorem at nibh. Curabitur nulla purus, feugiat id, elementum in, lobortis quis, pede. Vivamus sodales adipiscing sapien. Vestibulum posuere nulla eget wisi.</p>
+      <p>Integer volutpat ligula eget enim. Suspendisse vitae arcu. Pellentesque nibh felis, eleifend id, commodo in, interdum vitae, leo. Praesent eu elit. Ut eu ligula. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos hymenaeos. Maecenas elementum augue nec nisl. Proin auctor lorem at nibh. Curabitur nulla purus, feugiat id, elementum in, lobortis quis, pede. Vivamus sodales adipiscing sapien. Vestibulum posuere nulla eget wisi.</p>
+    </div>
+
+    <table class="w-100 profile-detail__link-table">
+      <tr class="sr-only">
+        {% for name in links %}
+          <th>{{ name }}</th>
+        {% endfor %}
+      </tr>
+      <tr class="d-flex">
+        {% for name in links %}
+          <td>
+            <strong class="f2 d-block">{{ name }}</strong>
+            {% if name == 'Links' %}
+              <ul class="f2">
+                <li>
+                  <a href="#" class="link-underline">Curriculum Vitae</a>
+                </li>
+                <li>
+                  <a href="#" class="link-underline">aaronsmithisthebest.com</a>
+                </li>
+              </ul>
+            {% else %}
+              {% include 'partials/social-links.twig' with {
 								items: social_links
 							} %}
-						{% endif %}
-					</td>
-				{% endfor %}
-			</tr>
-		</table>
-
-		<section
-			class="section" aria-labelledby="midd-profile-section-courses-label">
-			<!-- <h2 class="section__title" id="midd-profile-section-courses-label">Courses Taught</h2> -->
-			{% include 'paragraphs/course-list.twig' with {
+            {% endif %}
+          </td>
+        {% endfor %}
+      </tr>
+    </table>
+
+    <section class="section" aria-labelledby="midd-profile-section-courses-label">
```

Comment:
I don't think we need this `<section>` element since the course list template starts with one.

Reply:
You're right, I'll remove that tag.